go-fuzz-build/main.go

```diff
@@ -24,6 +24,7 @@ import (
 )
 
 var (
+	flagTag  = flag.String("tags", "", "a list of build tags to consider satisfied during the build.")
```

Comment:
Proper way of doing it is using: https://godoc.org/golang.org/x/tools/go/buildutil#TagsFlag